index.html

```diff
@@ -1238,6 +1238,318 @@ <h2>Evidence</h2>
+		"name": "Mr John Doe",
+		"address": {
+                   "streetAddress": "10 Rue de Chose",
+                   "postalCode": "98052",
+                   "addressLocality": "Paris, France"
+                },
+		"birthDate": "2001-01-01",
+		"gender": "http://schema.org/Male",
+		"birthPlace": {
+                   "streetAddress": "36 Rue de Different",
+                   "postalCode": "102435",
+                   "addressLocality": "Paris"
+                   "addressCountry": "FR"
+                },
+
+		"nationality": "FR",
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+
+   </section>
+
+    <section>
+       <h2>Subject Passes the Verifiable Credential to a Holder</h2>
+
+     <p>
+Normally verifiable credentials will be presented to verifiers by the subject. However in some cases, the subject may need to pass the whole or part of a verifiable credential to a holder. The data model allows for both cases as follows. </p>
+
+<p>
+If only the subject is allowed to present the verifiable credential to a verifier, the issuer MUST insert the "subject only" Terms of Use into the verifiable credential, as defined below.  </p>
+
+<p>
+If the subject is not forbidden by the issuer to pass the whole or part of a verifiable credential to a holder, for example, a patient may pass a prescription verifiable credential to a relative for the relative to present it to a pharmacist for dispensing, then the subject MUST issue a verifiable credential to the holder containing the claim properties that are being passed on, as described below. </p>
+
+   <section>
+
+   <h3> 'Subject Only' Terms of Use </h3>
+
+  <p>
+The Subject Only Terms of Use states that a verifiable credential MUST only be presented to a verifier by the subject. If a verifier is presented with a verifiable credential containing the Subject Only Terms of Use, by anyone other than the subject, it MUST refuse to accept it.</p>
+
+        <pre class="example nohighlight" title="Subject Only termsOfUse property by an Issuer">
+{
+  "id": "http://dmv.example.gov/credentials/3732",
+  "type": ["VerifiableCredential", "ProofOfAgeCredential"],
+  "issuer": "https://dmv.example.gov/issuers/14",
+  "issued": "2010-01-01T19:73:24Z",
+  "claim": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "ageOver": 21
+  },
+  <span class="highlight">"termsOfUse": [{
+    "type": "SubjectOnly",
+    }]
+  }</span>,
+  "proof": { ... }
+}
+        </pre>
+
+       </section>
+       
+  <section>
+   <h3> Passing on a Verifiable Credential </h3>
+
+   <p>
+
+When the subject passes a verifiable credential to a
+holder, the subject SHOULD issue a new verifiable credential to the holder in which: 
+the issuer is the subject, 
+the subject is the holder to whom the verifiable credential is being passed, 
+and the claim contains the properties that are being passed on. In addition, the holder creates a verifiable presentation that contains these two
```

Comment:
"the claim contains the properties that are being passed on" creates an enormous amount of complexity when processing credentials that are handed off... We should probably make it an all or nothing thing for v1.0.

Again, fine with pulling this in as long as we can change this later.